festim/geometry/mesh.py

```diff
@@ -0,0 +1,57 @@
+import ufl
+
+
+class Mesh:
+    """
+    Mesh class
+    Attributes:
+        mesh (dolfinx.Mesh): the mesh
+        volume_markers (dolfinx.MeshTags): markers of the mesh cells
+        surface_markers (dolfinx.MeshTags): markers of the mesh facets
+        dx (dolfinx.Measure):
+        ds (dolfinx.Measure):
+    """
+
+    def __init__(
+        self,
+        mesh=None,
+        volume_markers=None,
+        surface_markers=None,
+        subdomains=[],
+        **kwargs
```

Comment:
What are the `kwargs` for here? doesn't seem to be used anywhere else in the `__init__`